lib/src/logic.dart

```diff
@@ -648,6 +652,10 @@ class Logic {
         (startIndex < 0) ? width + startIndex : startIndex;
     final modifiedEndIndex = (endIndex < 0) ? width + endIndex : endIndex;
 
+    if (width == 1 && endIndex == 0 && modifiedEndIndex == modifiedStartIndex) {
```

Comment:
Shouldn't this be `modifiedEndIndex == 0`?  What if `endIndex` is -1 on a 1-bit signal?

Reply:
in that case BusSubset will throw an exception during initialization